test/controllers/names_controller_test.rb

```diff
@@ -266,7 +260,7 @@ def test_pagination_page2
   def test_pagination_letter
```

Comment:
It's great to get rid of the raw sql, but do these changes belong in this PR?

Reply:
I can move to another PR, no prob